testing/unit/rpc/test_block_info.py

```diff
@@ -389,6 +389,23 @@
     assert 'Header signature is not of a blockchain token type.' == error.value.message
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize('non_existing_start', ['a'*128])
+async def test_list_blocks_with_non_existing_start(request_, non_existing_start):
```

Comment:
@Alladin9393, add test for parameter ids.

Reply:
@anastasiia-bilova, done.